spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,32 +4,61 @@
 context 'when switching phases' do
   subject { Season.current }
 
-  describe '#fake_application_phase' do
-    it 'timeshifts the application phase to today' do
-      Timecop.travel(Season.current.applications_close_at - 1.month) do
-        Season::PhaseSwitcher.fake_application_phase
-        expect(subject).to be_application_period
+  describe 'destined' do
+
+    it 'sends the params to a phase setting method' do
+      phase = 'fake_application_phase'
+      Season::PhaseSwitcher.destined(phase)
+      expect(subject).to_not be_nil
+    end
+
+    context 'when directed to the #fake_application_phase' do
+      it 'sets date range so that the season is open for application submissions' do
+        phase = 'fake_application_phase'
+        Timecop.travel(Season.current.applications_close_at - 1.month) do
+          Season::PhaseSwitcher.destined(phase)
+          expect(subject).to_not be_nil
+          expect(subject).to be_application_period
+        end
       end
     end
-  end
 
-  describe '#fake_coding_phase' do
-    it 'timeshifts the coding phase to today' do
-      Timecop.travel(Season.current.ends_at - 1.week) do
-        Season::PhaseSwitcher.fake_coding_phase
-        expect(subject).to be_started
+    context 'when directed to the #fake_coding_phase' do
+      it 'sets dates so that the summer of code is currently happening' do
+        phase = 'fake_coding_phase'
+        Timecop.travel(Season.current.ends_at - 1.week) do
```

Comment:
That made all the difference. 🙈 
Second try,  to only test the public interface. To understand what is happening, I needed to work on a lower abstraction level than your instructions. I can change that later, unless you are tired of it...
The `expect(Season).to receive(:current).and_return(season)`did not want to return `season`, so I just added a var. 
What I tried to accomplish: see if the season receives the call with the phase, by checking roughly if something in the output changed. 
And now I realised why I added the Timecop: when the test run during the same phase as the fake-phase. I solved it not by adding a test in another phase. 
wdyt?